src/CONST.js

```diff
@@ -756,6 +756,13 @@ const CONST = {
         },
     },
     MICROSECONDS_PER_MS: 1000,
+    RBR: {
+        PENDING_ACTION: {
```

Comment:
IDK if this nesting is too much. It could also be:
```javascript
RBR_PENDING_ACTION: {
    ADD: 'add',
    DELETE: 'delete',
    UPDATE: 'update',
}
```
I also wasn't sure if it should be plural, e.x `PENDING_ACTIONS`.

Reply:
I like `RED_BRICK_ROAD_PENDING_ACTION` and keep it singular.

Reply:
To clarify, which do you prefer?

1. `CONST.RED_BRICK_ROAD.PENDING_ACTION.UPDATE`
2. `CONST.RED_BRICK_ROAD.RED_BRICK_ROAD_PENDING_ACTION.UPDATE`
3. `CONST.RED_BRICK_ROAD_PENDING_ACTION.UPDATE`

Reply:
3 👍